docs/pbc/all/tax-management/202400.0/base-shop/vertex/install-vertex.md

```diff
@@ -39,24 +39,31 @@ Add the following config to `config/Shared/config_default.php`:
 use Generated\Shared\Transfer\ConfigureTaxAppTransfer;
 use Generated\Shared\Transfer\DeleteTaxAppTransfer;
 use Generated\Shared\Transfer\SubmitPaymentTaxInvoiceTransfer;
+use Spryker\Shared\TaxApp\TaxAppConstants;
+use Spryker\Zed\OauthAuth0\OauthAuth0Config;
 
 // ...
 
-$config[MessageBrokerConstants::MESSAGE_TO_CHANNEL_MAP] =
-$config[MessageBrokerAwsConstants::MESSAGE_TO_CHANNEL_MAP] = [
+$config[TaxAppConstants::OAUTH_PROVIDER_NAME] = OauthAuth0Config::PROVIDER_NAME;
+$config[TaxAppConstants::OAUTH_GRANT_TYPE] = OauthAuth0Config::GRANT_TYPE_CLIENT_CREDENTIALS;
+$config[TaxAppConstants::OAUTH_OPTION_AUDIENCE] = 'aop-app';
```

Comment:
OAuth configuration is defined after messagebroker configuration, let's keep it consistent with the ordering of configuration in our demoshops.

Reply:
OAuth lines are split into groups so provider name for all configs is defined in one place, grant type in another and so on. Is that OK to have it as a single group here?